java-squid/src/main/java/org/sonar/java/model/InternalVisitorsBridge.java

```diff
@@ -50,6 +51,7 @@
 
   private final List<JavaFileScanner> scanners;
   private final SonarComponents sonarComponents;
+  private final boolean symbolicExecutionEnabled;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "symbolicExecutionEnabled" private field. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS1068)